tests/baselines/reference/jsdocLink1.baseline

```diff
@@ -0,0 +1,191 @@
+[
+  {
+    "marker": {
+      "fileName": "/tests/cases/fourslash/jsdocLink1.ts",
+      "position": 189,
+      "name": ""
+    },
+    "quickInfo": {
+      "kind": "function",
+      "kindModifiers": "",
+      "textSpan": {
+        "start": 189,
+        "length": 2
+      },
+      "displayParts": [
+        {
+          "text": "function",
+          "kind": "keyword"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "CC",
+          "kind": "functionName"
+        },
+        {
+          "text": "(",
+          "kind": "punctuation"
+        },
+        {
+          "text": ")",
+          "kind": "punctuation"
+        },
+        {
+          "text": ":",
+          "kind": "punctuation"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "void",
+          "kind": "keyword"
+        }
+      ],
+      "documentation": [
+        {
+          "text": "",
+          "kind": "text"
+        },
+        {
+          "text": "\n",
+          "kind": "lineBreak"
+        },
+        {
+          "text": "{@link C}",
+          "kind": "link",
+          "name": {
+            "fileName": "/tests/cases/fourslash/jsdocLink1.ts",
+            "textSpan": {
+              "start": 26,
+              "length": 1
+            }
+          },
+          "target": {
+            "fileName": "/tests/cases/fourslash/jsdocLink1.ts",
+            "textSpan": {
+              "start": 0,
```

Comment:
What span is this?  I think it's relative to the start of the target file?

Reply:
Yes, `class C {}` just happens to be the first thing in the file for this test.